docs/docs/widgets/link.md

```diff
@@ -47,15 +46,15 @@ Following actions of link component can be controlled using the component specif
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Exposed Variables
 
 There are currently no exposed variables for the component.
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## General
 ### Tooltip
```

Comment:
Hey @AsifQamar here on line no. 62 you need to change the widget to the component.